.editorconfig

```diff
@@ -0,0 +1,23 @@
+# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
> Someone is going to change their settings from the deafult

That's a good point. The logical conclusion to that is that we should specify _all_ editorconfig settings, not just some. See https://github.com/dotnet/roslyn/blob/master/.editorconfig